neuralforecast/tsdataset.py

```diff
@@ -199,6 +199,53 @@ def update_dataset(dataset, future_df):
 
         return updated_dataset
 
+    @staticmethod
+    def trim_dataset(dataset, left_trim: int = 0, right_trim: int = 0):
+        """Trim temporal information from a dataset."""
```

Comment:
"""
Trim temporal information from a dataset.
Keeping window indexes [t+left:t-right] for all series.
"""